light-cli/src/commands/airdrop/index.ts

```diff
@@ -0,0 +1,102 @@
+import { Args, Command, Flags } from "@oclif/core";
+import { PublicKey } from "@solana/web3.js";
+import {
+  CustomLoader,
+  generateSolanaTransactionURL,
+  getConnection,
+} from "../../utils/utils";
+import { getOrCreateAssociatedTokenAccount, mintTo } from "@solana/spl-token";
+import { ADMIN_AUTH_KEYPAIR, MINT } from "@lightprotocol/zk.js";
+
+class AirdropCommand extends Command {
+  static description = "Perform a native Solana or SPL airdrop to a user";
+
+  protected finally(_: Error | undefined): Promise<any> {
+    process.exit();
+  }
+
+  static flags = {
+    amount: Flags.integer({
+      char: "a",
+      description: "The amount to airdrop",
+      required: true,
+    }),
+    token: Flags.string({
+      char: "t",
+      description: "The token to airdrop",
+      required: true,
+    }),
+  };
+
+  static examples = [
+    `$ light airdrop --token SOL --amount 2000000000 <userPublicKey>`,
```

Comment:
The provided amounts seem to be in lamports not sol.
I think lamports are impractical for an airdrop command.
pls add decimal conversion
you can import the decimal conversion from zk,js

Reply:
Done!